src/modules/nodejs.rs

```diff
@@ -57,11 +57,15 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 _ => None,
             })
             .map(|variable| match variable {
-                "version" => nodejs_version
-                    .deref()
-                    .as_ref()
-                    .map(|version| version.trim())
-                    .map(Ok),
+                "version" => Some(Ok(VersionFormatter::new(config.version_format)
+                    .ok()?
+                    .format_version(
+                        &nodejs_version
```

Comment:
Note: this is not working correctly in this revision because node.js has `v` as a prefix, and I'm not printing structured versions here.

Reply:
Okay, this has been addressed.